src/rpcq.lisp

```diff
@@ -313,7 +313,7 @@ LIMITATIONS:
                    ,@(when deprecated-by
                        `(:initarg ,(intern (symbol-name deprecated-by) :keyword)))
 
-                   :reader ,(accessor slot-name)
+                   :accessor ,(accessor slot-name)
```

Comment:
Personally, I like the idea of rpc message slots being read-only, but I haven't really used the API at all, so if it's a pain-point or you feel it will significantly improve the configurable protoquil PR, I guess go for it.

Reply:
I can't think of a good reason *why* they should be read-only. The use-case I would like to enable is piece-wise construction of a message. Do you think there is a down-side to read/write (other than *mutation is bad!* :))

Reply:
Pretty much just "mutation is bad".  :-)